src/main/java/org/apache/maven/shared/utils/xml/Xpp3DomUtils.java

```diff
@@ -148,15 +148,15 @@ private static boolean isMergeChildren( Xpp3Dom dominant )
     }
 
     /**
-     * @param str The string to be checked.
-     * @return <code>true</code> in case string is empty <code>false</code> otherwise.
+     * @deprecated use <code>String.isBlank()</code> (Java 11+) 
```

Comment:
`String#isBlank()` is *not* a replacement because it will cause a NPE if null.

Reply:
fixed